src/dnstable_lookup.c

```diff
@@ -236,6 +245,9 @@ main(int argc, char **argv)
 		case 'J':
 			g_Json = true;
 			break;
+		case 'n':
```

Comment:
Is missing corresponding change to getopt()

Reply:
Fixed.